styleguide/derek/layouts/additional-resources.ejs

```diff
@@ -1,29 +1,21 @@
 <!-- Additional Resources: 33/34/33 -->
-<div class="container half-padding-full">
-  <div class="row row-eq-height">
-    <div class="col-sm-4 panel-flex">
-      <%- partial('../_partials/cards/additional-resource.ejs') %>
-    </div>
-    <div class="col-sm-4 panel-flex">
-      <%- partial('../_partials/cards/additional-resource.ejs', {
-        type: 'video'
-      }) %>
-    </div>
-    <div class="col-sm-4 panel-flex">
-      <%- partial('../_partials/cards/additional-resource.ejs', {
-        type: 'rss',
-        desc: 'consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean m'
-      }) %>
-    </div>
-  </div>
-  <div class="row row-eq-height">
-    <div class="col-sm-6 panel-flex">
-      <%- partial('../_partials/cards/additional-resource.ejs', {type: 'news'}) %>
-    </div>
-    <div class="col-sm-6 panel-flex">
-      <%- partial('../_partials/cards/additional-resource.ejs', {
-        type: 'support'
-      }) %>
+<div class="half-padding-full bg-light-gray">
+  <div class="container">
+    <div class="row row-eq-height">
+      <div class="col-sm-4 panel-flex">
+        <%- partial('../_partials/cards/additional-resource.ejs') %>
+      </div>
+      <div class="col-sm-4 panel-flex">
+        <%- partial('../_partials/cards/additional-resource.ejs', {
+          type: 'video'
+        }) %>
+      </div>
+      <div class="col-sm-4 panel-flex">
+        <%- partial('../_partials/cards/additional-resource.ejs', {
+          type: 'rss',
+          desc: 'consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean m'
+        }) %>
+      </div>
     </div>
   </div>
 </div>
```

Comment:
This is because of the weird padding issues we're having. Hoping to be able to tackle that later this sprint.